usersession/userd/launcher.go

```diff
@@ -123,6 +130,126 @@ func (s *Launcher) OpenURL(addr string, sender dbus.Sender) *dbus.Error {
 	return nil
 }
 
+// OpenDesktopEntryEnv implements the 'OpenDesktopEntryEnv' method of the 'io.snapcraft.Launcher'
+// DBus interface.
+func (s *Launcher) OpenDesktopEntryEnv(desktop_file_id string, env []string, sender dbus.Sender) *dbus.Error {
```

Comment:
I think that as we add more methods to `snap userd` we should be documenting them. @degville where do you think such documentation should live?

As for this method, I'm unfamiliar with desktop file IDs, could the function documentation at least indicate what those are? Is the return value sufficient for shells? Do they need additional features, such as PID tracking?